Source/Filter Effects/feColourMatrix/SvgColourMatrix.cs

```diff
@@ -87,7 +87,7 @@ public override void Process(ImageBuffer buffer)
                     };
                     break;
                 default: // Matrix
-                    var parts = this.Values.Replace("  ", " ").Split(new char[] { ' ', '\t', '\n', '\r', ',' });
+                    var parts = this.Values.Replace("  ", " ").Split(new char[] { ' ', '\t', '\n', '\r', ',' }, StringSplitOptions.RemoveEmptyEntries);

```

Comment:
I think replacement is also unnecessary.
```cs
var parts = this.Values.Split(new char[] { ' ', '\t', '\n', '\r', ',' }, StringSplitOptions.RemoveEmptyEntries);
```

Reply:
Yes, pushed change